### PR TITLE
Include docs for web app and algolia cli

### PR DESCRIPTION
**Pull request by silentlight**
- Add docs for web app
- Add docs for Algolia CLI command

### CODE REVIEW

**Review by Necmttn — APPROVED** [review]
LGTM

**Review by zurgl — APPROVED** [review]
LGTM

File: apps/docs/pages/web-app/how-kafe-serves-content.mdx [hunk 1]
```diff
@@ -28,8 +28,59 @@ Due to Vercel's [limitation of serverless function sizes](https://github.com/ver
...
+
+Currently, most of the guides on Kafé are single page guides.
+Our platforms supports multiple page guides as well.
+As an author of a guide you will be presented with an option to select weather you want to create a single-page guide or a multi-page guide when using BuilderDAO CLI (**builderdao tutorial init**)
+
+![cli guide type selection](/assets/images/cli-page-type-selection.png)
+
+## Markdown parsing
+
+Kafé uses [MDX](https://mdxjs.com/docs/) what makes it east to use JSX in your markdown content.
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/how-kafe-serves-content.mdx]: > Kafé uses [MDX](https://mdxjs.com/docs/) which makes it easy to use JSX in your markdown content.

File: apps/docs/pages/web-app/how-kafe-serves-content.mdx [hunk 2]
```diff
@@ -28,8 +28,59 @@ Due to Vercel's [limitation of serverless function sizes](https://github.com/ver
...
+
+### Frontmatter
+
+Eveny _.mdx_ file consists of frontmatter which describes the content that is inside of the file.
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/how-kafe-serves-content.mdx]: > Every _.mdx_ file contains frontmatter which describes the content inside of the file.

File: apps/docs/pages/web-app/summary.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Web app
+---
+
+import Authors, { Author } from '@app/components/authors';
+
+# Web App
+
+Web App is a main application users will use to interact with Kafé.
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/summary.mdx]: > The Kafé web app is how users interact with the on-chain program.

File: apps/docs/pages/web-app/summary.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+It’s main purpose is to serve content to users. Using web app users will be able to perform 4 main actions:
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/summary.mdx]: > It’s main purpose is to serve content to users. Using the web app, users will be able to perform 4 main actions:

File: apps/docs/pages/web-app/using-solana-wallet.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Technology stack
+---
+
+import Authors, { Author } from '@app/components/authors';
+
+# Using Solana wallet
+
+Kafé is built on top of Solana blockchain. In order to interact with Solana and therefore with Kafé, every users needs to have a wallet. There are many different wallets you can use:
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/using-solana-wallet.mdx]: > The Kafé program is built on the Solana blockchain. In order to interact with Solana and therefore with Kafé, every user needs to have a wallet, and SOL tokens. There are many different wallets you can use:

File: apps/docs/pages/web-app/using-solana-wallet.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+- Phantom
+- Solflare
+- Sollet
+- Glow Wallet
+- Torus Wallet
+- Ledger Wallet
+  Our web app in order to connect to those wallets uses great npm package called Solana Wallet Adapter. It comes with set of React components and utility functions to make connection to all above wallets seamless.
```
Comment (pyr0gan) [inline apps/docs/pages/web-app/using-solana-wallet.mdx]: > To connect to those wallets, Kafé uses an NPM package called Solana Wallet Adapter. It comes with a set of React components and utility functions to make the connection process seamless.

**Review by pyr0gan — APPROVED** [review]
@silentlight Just suggesting a couple of small tweaks to improve readability 😀 Otherwise, looks good!

**Pull request merged**